### PR TITLE
Add a Kotlin DSL sample into README

### PR DESCRIPTION
**Pull request by blindpirate**

### CODE REVIEW

**Comment by blindpirate** [comment]
This closes https://github.com/etiennestuder/gradle-rocker-plugin/issues/7

File: README.md [hunk 1]
```diff
@@ -79,7 +79,28 @@ rocker {
...
+
+```
+import nu.studer.gradle.rocker.RockerConfig
+
+plugins {
+    id("nu.studer.rocker") version "1.0.1"
+    id("java")
+}
+
+configure<NamedDomainObjectContainer<RockerConfig>>() {
```
Comment (eskatos) [inline README.md:95]: The usage of the plugins block above make it possible to use type safe generated accessors.
Also, the import above is not needed then.

```suggestion
rocker {
```

File: README.md [hunk 2]
```diff
@@ -79,7 +79,28 @@ rocker {
...
+    create("main") {
+        setTemplateDir(file("src/rocker"))
```
Comment (eskatos) [inline README.md:97]:
```suggestion
        templateDir = file("src/rocker")
```

File: README.md [hunk 3]
```diff
@@ -79,7 +79,28 @@ rocker {
...
+        setOutputDir(file("src/generated/rocker"))
```
Comment (eskatos) [inline README.md:98]:
```suggestion
        outputDir = file("src/generated/rocker")
```

File: README.md [hunk 4]
```diff
@@ -79,7 +79,28 @@ rocker {
...
+        setOptimize(true) // optional
```
Comment (eskatos) [inline README.md:99]:
```suggestion
        isOptimize = true // optional
```

File: README.md [hunk 5]
```diff
@@ -79,7 +79,28 @@ rocker {
...
+    }
+}
+
+extra["rockerVersion"] = "1.3.0"
```
Comment (eskatos) [inline README.md:103]: This is correct but could also be changed to:
```suggestion
val rockerVersion by extra("1.3.0")
```

Not related to this PR but why not provide this configuration as a proper property on the rocker extension?

**Comment by etiennestuder** [comment]
Thanks. This is now on master, pending 2.1.1 to show up on the plugin portal before the example works.

**Pull request closed without merging**